invenio/b2share/modules/b2deposit/b2share_marc_handler.py

```diff
@@ -157,6 +158,12 @@ def add_file_info(rec, form, email, sub_id, recid):
     else:
         fft_status = 'firerole: allow email "{0}"\ndeny all'.format(
             email)
+
+    if 'embargo_till' in form:
+        embargodate = parser.parse(form['embargo_till'])
+        embargodate = datetime.strftime(embargodate, '%Y-%m-%d')
+        fft_status = 'firerole: deny until "%s"\nallow any\n' % embargodate
```

Comment:
@emanueldima Seems like you are right, `deny until`  does at least not behave as I had expected. However, I'm not sure what is optimal, so it is good to have it on the test server so we can get some feedback now. I'll try to squeeze in some improvements before the weekend.